diff --git a/node_modules/vite-plugin-svgr/client.d.ts b/node_modules/vite-plugin-svgr/client.d.ts
index dbfc650..fcd6218 100644
--- a/node_modules/vite-plugin-svgr/client.d.ts
+++ b/node_modules/vite-plugin-svgr/client.d.ts
@@ -5,7 +5,7 @@ declare module "*.svg?react" {
   import * as React from "react";
 
   const ReactComponent: React.FunctionComponent<
-    React.ComponentProps<"svg"> & { title?: string }
+    React.ComponentProps<"svg"> & { title?: string; alt?: string }
   >;
 
   export default ReactComponent;
